agent/src/main/java/com/teamscale/jacoco/agent/upload/teamscale/DelayedTeamscaleMultiProjectUploader.java

```diff
@@ -20,6 +21,34 @@ public DelayedTeamscaleMultiProjectUploader(BiFunction<String, CommitInfo, IUplo
 		this.uploaderFactory = uploaderFactory;
 	}
 
+	/**
+	 * Checks if there is already a uploader present with the given project and commit. This is relevant to prevent
+	 * uploading coverage multiple times to the same location.
+	 */
+	public boolean projectAndCommitAlreadyRegistered(ProjectAndCommit projectAndCommit) {
+		return teamscaleUploaders.stream().anyMatch(uploader -> {
+			// Might happen during testing.
+			if (uploader == null) {
+				return false;
+			}
```

Comment:
Yes, this was done as a shortcut in `GitMultiProjectPropertiesLocatorTest#testNoErrorIsThrownWhenGitPropertiesFileDoesNotHaveAProject`. No longer relevant since the marked code snippet does not exist anymore.